packages/gatsby-adapter-netlify/src/file-cdn-url-generator.ts

```diff
@@ -1,4 +1,4 @@
-import crypto from "crypto"
+import { createHash } from "crypto"
```

Comment:
This caused some problems with 
```
Cannot find module '@babel/runtime/helpers/interopRequireDefault'
```
in lambdas, so easiest way was just to avoid having babel adding that